packages/itinerary-body/src/defaults/line-column-content.tsx

```diff
@@ -65,14 +65,18 @@ export default function LineColumnContent({
     }
   );
 
+  const routeNumber = typeof route === "string" ? route : route?.shortName;
+
   return (
     <S.LegLine>
       {!isDestination && <S.InnerLine mode={mode} routeColor={routeColor} />}
       <S.LineBadgeContainer>
         {/* TODO: This is a placeholder for a routebadge when we create the transit leg */}
         {!interline && !isDestination && transitLeg && (
           <RouteBadge
-            abbreviation={toRouteAbbreviation(parseInt(route, 10) || route)}
+            abbreviation={toRouteAbbreviation(
+              parseInt(routeNumber, 10) || routeNumber
```

Comment:
I think this will crash on a short name like "Circulator" or similar

Reply:
This already handles route short name such as "MAX Blue Line" correctly, see http://www.opentripplanner.org/otp-ui/?path=/story/itinerarybody-otp-ui--walk-transit-walk-itinerary&globals=locale:en-US.